sentry_sdk/integrations/django/__init__.py

```diff
@@ -358,9 +363,33 @@ def record_sql(sql, params, cursor=None):
         except Exception:
             pass
 
+    span = None
+
     if real_sql:
         with capture_internal_exceptions():
             hub.add_breadcrumb(message=real_sql, category="query")
+            span = hub.start_span(op="sql.query", description=real_sql)
```

Comment:
Would be nice to follow the [semantic conventions](https://github.com/opentracing/specification/blob/master/semantic_conventions.md) from OT here. That'd be `db.statement`. Or is this something that needs to be consistent with the `breadcrumbs` component?

Reply:
Renamed to db.statement